lib/ssl/src/ssl_connection.hrl

```diff
@@ -81,7 +81,8 @@
                         public_key_info      :: ssl_handshake:public_key_info() | 'undefined',
                         premaster_secret     :: binary() | secret_printout() | 'undefined',
                         server_psk_identity         :: binary() | 'undefined',  % server psk identity hint
-                        ticket_seed
+                        ticket_seed,
+                        ocsp_stapling_state = #{}
```

Comment:
See my comments above, we probably don't need the ocsp_stapling_state.

Reply:
> See my comments above, we probably don't need the ocsp_stapling_state.

Currently I store the ocsp status result in the state, if we will remove this, where should we store the result then?

Reply:
I think we will not need to store it, the result can be used directly when we verify the certificate chain.